src/board/stroke/stroke.tsx

```diff
@@ -1,57 +1,36 @@
 import { Polygon } from "sat"
-import { KonvaEventObject } from "konva/types/Node"
-import { simplifyRDP } from "drawing/simplify"
-import store from "redux/store"
-import { getHitboxPolygon, setSelectedShapes } from "board/hitbox/hitbox"
-import { LiveStroke, Stroke, ToolType } from "./types"
-import { BoardLiveStroke, isContinuous } from "./livestroke"
-import {
-    CANVAS_FULL_HEIGHT,
-    RDP_FORCE_SECTIONS,
-    RDP_EPSILON,
-} from "../../constants"
+import { getHitboxPolygon } from "board/hitbox/hitbox"
+import { Stroke, ToolType } from "./types"
 
-export class BoardStroke extends BoardLiveStroke implements Stroke {
+export class BoardStroke implements Stroke {
     /**
      * Create a new stroke from another Stroke instance
-     * or a LiveStroke when pageId is given.
      */
-    constructor(stroke: Stroke, pageId?: string) {
-        super()
-        // import properties from livestroke
+    constructor(stroke: Stroke) {
```

Comment:
Maybe no, lets keep the livestroke logic contained, I dont want it in the main stroke logic since it is unrelated and only used once when a livestroke is finished